sprockets/http/__init__.py

```diff
@@ -77,8 +90,16 @@ def run(create_application, settings=None, log_config=_unspecified):
 
     port_number = int(app_settings.pop('port', os.environ.get('PORT', 8000)))
     num_procs = int(app_settings.pop('number_of_procs', '0'))
-    server = runner.Runner(create_application(**app_settings))
+    app = create_application(**app_settings)
+
+    if sentry_sdk is not None:
+        sentry_sdk.init(
+            integrations=_sentry_integrations,
+            release=app.settings.get('version'),
```

Comment:
Based on our general usage, I don't think `version` and `environment` is available at this level? They are normally in the created application settings.

Reply:
nvm, it's a `.` vs. and `_`...saw `app.settings` and thought it was `app_settings`